InvenTree/part/views.py

```diff
@@ -688,6 +693,18 @@ def post(self, request, *args, **kwargs):
             except AttributeError:
                 pass
 
+            # Create part parameters
```

Comment:
I think it would be a cleaner approach to use the save method of the Part model to add default parameter values, rather than manually performing the action here.

That way, the parameters will also be added if a Part is created via the API or the admin interface, or wherever.

In the save() method for the Part object, you can check if the object is being created (first call to save) or updated (subsequent calls)

Reply:
@SchrodingersGat This is a smart idea, I like it.
I don't have time to do this now but I'll work on it asap.

Thanks for taking a look at this :smile: